explorer/app/viewModelBuilders/anvil-catalog/common/viewModelBuilders.ts

```diff
@@ -0,0 +1,83 @@
+import {
+  getStudyName,
+  getConsortia,
+  getDisease,
+  getDataType,
+  getIndication,
+  getParticipantCount,
+  getStudyDesign,
+} from "../../../apis/anvil-catalog/common/transformers";
+import * as C from "../../../components";
+import { AnvilSourceItem } from "../../../apis/anvil/common/entities";
+
+const createColumn = (
+  value: string | number = ""
+): React.ComponentProps<typeof C.Text> => ({
+  children: value,
+  customColor: "ink",
+  variant: "text-body-400",
+});
+
+/**
+ * Build props for study name cell component from the given study in the Anvil catalog.
+ * @param source - Study row in the Anvil catalog TSV.
+ * @returns Model to be used as props for the platform cell.
+ */
+export const buildStudyName = (
+  source: AnvilSourceItem
+): React.ComponentProps<typeof C.Text> => createColumn(getStudyName(source));
```

Comment:
Replace usage of `Text` with `Cell`, here and throughout (and also in the view model builder). Ideally we'd just call `ViewModelBuilder.getStudyName()` here (and we can then also remove the `createColumn` function above).